### PR TITLE
migrating code from js to ts

### PR DESCRIPTION
**Pull request by rameshvishnoi90904**

### CODE REVIEW

**Comment by ftonato** [comment]
This _pull request_ is not concluded, right?

**Comment by rameshvishnoi90904** [comment]
@ftonato This is concluded/

**Comment by ftonato** [comment]
I'm seeing your' update and a have some questions about this PR:

- Why you changed the variables that has used `const` and `let` to `var`?

**Comment by ftonato** [comment]
I'm seeing your' update and a have some questions about this PR:

- Why did you change the variables that has used `const` and `let` to `var`?
- Why don't you followed the indentation from our original code?
- Why did you change the strings accents to Unicode?

File: cli.js [hunk 1]
```diff
@@ -1,11 +1,8 @@
 #! /usr/bin/env node
-"use strict";
-exports.__esModule = true;
 var rbr = require('./index.ts');
 var chalk = require('chalk');
 var ora = require('ora');
-var escape = require("entities");
-// const escape = require('entities');
+var escape = require('entities');
```
Comment (ftonato) [inline cli.js:5]: Why did you change the variables that has used `const` and `let` to `var`?

File: cli.js [hunk 2]
```diff
@@ -1,80 +1,72 @@
...
+// const escape = require('entities');
+var domain = process.argv[2] || '';
 function parse(data) {
-  if (data.available) {
-    console.log(chalk.green(`Domínio ${chalk.bold(data.fqdn)} está disponível`));
-  } else {
-    console.log(chalk.red(`Domínio ${chalk.bold(data.fqdn)} não está disponível`));
-    if (data.reason) {
-      console.log(chalk.red.bold(escape.decodeHTML(data.reason)));
+    if (data.available) {
```
Comment (ftonato) [inline cli.js:8]: Why don't you followed the indentation from our original code?

File: cli.js [hunk 3]
```diff
@@ -1,80 +1,72 @@
...
+        console.log(chalk.green("Dom\u00EDnio " + chalk.bold(data.fqdn) + " est\u00E1 dispon\u00EDvel"));
+    }
+    else {
+        console.log(chalk.red("Dom\u00EDnio " + chalk.bold(data.fqdn) + " n\u00E3o est\u00E1 dispon\u00EDvel"));
```
Comment (ftonato) [inline cli.js:12]: Why did you change the strings accents to Unicode?

**Review by ftonato — CHANGES_REQUESTED** [review]

**Pull request closed without merging**